app/src/main/java/de/westnordost/streetcomplete/quests/toilets_fee/ToiletsFeeForm.java

```diff
@@ -0,0 +1,27 @@
+package de.westnordost.streetcomplete.quests.toilets_fee;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import de.westnordost.osmapi.map.data.OsmElement;
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class ToiletsFeeForm extends YesNoQuestAnswerFragment
+{
+
+	@Override public View onCreateView(LayoutInflater inflater, ViewGroup container,
+							 Bundle savedInstanceState)
+	{
+		View view = super.onCreateView(inflater, container, savedInstanceState);
+		setTitle();
+		return view;
+	}
+
+	private void setTitle()
+	{
+		setTitle(R.string.quest_toiletsFee_title);
+	}
+}
```

Comment:
You do not need a form, just create a YesNoQuestAnswerFragment in the quest and set the title from there (`setTitle(int titleId)` is public)